libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/ApiVersionStrings.java

```diff
@@ -39,10 +39,22 @@
 
     public static final String VERSION_NUMBER = "v36.0";
     public static final String API_PREFIX = "/services/data/";
-    public static final String BASE_PATH = API_PREFIX + getVersionNumber(SalesforceSDKManager.getInstance().getAppContext());
```

Comment:
Calling getInstance() too soon will throw an exception, so it's a bad idea to call it from a member initializer.